pkg/virt-launcher/virtwrap/network/selinux_context_executor.go

```diff
@@ -0,0 +1,94 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+
+package network
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+	"syscall"
+
+	"github.com/opencontainers/selinux/go-selinux"
+
+	kvselinux "kubevirt.io/kubevirt/pkg/virt-handler/selinux"
+)
+
+const virtHandlerSELinuxLabel = "system_u:system_r:spc_t:s0"
+
+type SELinuxContextExecution struct {
+	launcherPID  int
+	cmdToExecute *exec.Cmd
+}
+
+func (ce *SELinuxContextExecution) Execute() error {
+	if isSELinuxEnabled() {
```

Comment:
I think this fits in the object state. It can be set as part of the constructor.